src/emulator/auth/handlers.ts

```diff
@@ -36,6 +36,49 @@ export function registerHandlers(
     const state = getProjectStateByApiKey(apiKey);
 
     switch (mode) {
+      case "recoverEmail": {
+        const EXPIRED_LINK_ERROR = {
+          error: `Your request to revert your email has expired or the link has already been used.`,
+          instructions:
+            "Try reverting using this link again. If you're still unsuccessful. You can edit the email in the Emulator UI.",
```

Comment:
Not a big fan of this message. Any ideas are welcome.

Reply:
```suggestion
            "If you're trying to test the reverting email flow, try changing the email again to generate a new link.",
```

Reply:
Thanks! Done.